resource-validator/src/main/scala/com/broadinstitute/dsp/resourceValidator/DataprocWorkerChecker.scala

```diff
@@ -53,27 +46,37 @@ object DataprocWorkerChecker {
 
             isAnomalyDetected match {
```

Comment:
Nit: `if (isAnomalyDetected) {...}` may be simpler

Reply:
I like match a bit better style-wise for large blocks, and if looks repetitive since there are so many ifs inside the match already

Reply:
I respect your taste 😊